### DISCUSSION TITLE
Exclude two dirs for compare

### DISCUSSION THREAD

**Discussion by Pok4** [WinMerge/winmerge#1239] [Q&A] [answered]
Hi guys...
I have a problem and i don't know what to do with the filters..
I want the folders:
**vendor/
forum/**
to not included in compare. 

What to do ? I don't want this folder to show in the gui too...
Please, give me a advice :)

Sorry for my english ..

**Comment by sdottaka** [comment, marked as answer]
Create and apply a file filter as follows:

~~~
name: #1239
desc: #1239

def: include

d: \\forum$
d: \\vendor$
~~~

![image](https://user-images.githubusercontent.com/98126/157557095-780ebb49-eb94-4dc9-a1a2-763214d5bc27.png)
![image](https://user-images.githubusercontent.com/98126/157557206-910e92e2-41df-4762-adaa-556630ffe72b.png)
![image](https://user-images.githubusercontent.com/98126/157557274-9da42a9d-8e9a-4e75-a849-ce9993d8f9f8.png)
![image](https://user-images.githubusercontent.com/98126/157557368-4b5ce7ed-5141-40c2-9ae6-3de3775bae53.png)

![image](https://user-images.githubusercontent.com/98126/157557740-ef24edcc-d6cc-4dab-84c8-83cdd5548a44.png)

![image](https://user-images.githubusercontent.com/98126/157557844-76ecafec-8665-4f02-a001-d605da12fa83.png)

**Comment by Pok4** [comment]
Thank you! It works! I do it by myself on the same way.. I just read a couple of threads in stackoverflow and documentation on the site.
Its very useful tool!
